mobify/astro-scaffold
# Hyb 694 error modal

Error Modal Scaffold

JIRA: [**[HYB-694] As a scaffold user, I want a no internet / error screen**](https://mobify.atlassian.net/browse/HYB-694)
Linked PRs: **None**

## Changes
- Error modals appear when navigating without internet
- Error modals reveal back button when the current view can navigate
- Error modal content is customizable through `errorConfig.js`

## How to test-drive this PR
- Run app without internet: error modal should appear
- Leave internet off, tap retry: error modal should reappear
- Turn on internet, tap retry: modal should dismiss & tab/drawer items should be loaded
- Navigate a level deeper, turn off internet and navigate: back button should appear when possible to navigate back
- Open cart without internet: error modal should appear & back button should dismiss both cart & error modal

## TODOS:
